### ISSUE TITLE
Upload failed.

### ISSUE THREAD

**Issue by Feelingly** [mgp25/Instagram-API#2564]
### Before submitting an issue make sure you have:
- [x] [Searched](https://github.com/mgp25/Instagram-API/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [x] [Read the FAQ](https://github.com/mgp25/Instagram-API/wiki/FAQ)
- [x] [Read the wiki](https://github.com/mgp25/Instagram-API/wiki)
- [x] [Reviewed the examples](https://github.com/mgp25/Instagram-API/tree/master/examples)
- [x] [Installed the api using ``composer``](https://github.com/mgp25/Instagram-API#installation)
- [x] [Using latest API release](https://github.com/mgp25/Instagram-API/releases)

### Purpose of your issue?
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [ ] Question
- [ ] Other

---

### The following sections request more details for particular types of issues, you can remove any section (the contents between the triple ---) not applicable to your issue.

---

### For a *bug report*, you must include *code* that will replicate the error, and the *error log/traceback*.

My current code:

```php
<?php
// Very important tho.
set_time_limit(0);
date_default_timezone_set('UTC');
require __DIR__.'/vendor/autoload.php';

// Settings:
$debug = false;
$truncatedDebug = false;
$ig = new \InstagramAPI\Instagram($debug, $truncatedDebug);

// Sets arguments as variables to make it easier.
$username = $argv[1];
$password = $argv[2];
$picture = $argv[3];
$caption = $argv[4];

// Useless but just needed ;)
echo "\n[?] Attempting to login with profile: $username\n";
echo "\n[?] Username: $username & Password: $password & Picture: $picture and caption: $caption\n";

if (strpos( $picture, ".mp4" ) == true) {
    echo "\n[?] This is a video, attempting to upload the video (.mp4).";
	// Login to the specified account.
	try {
	    $ig->login($username, $password);
	} catch (\Exception $e) {
	    echo "\n[!] An error has occured when attempting to login: ".$e->getMessage()."\n";
	    exit(0);
	}
	try {
	    $video = new \InstagramAPI\Media\Video\InstagramVideo($picture);
	    $ig->timeline->uploadVideo($video->getFile(), ['caption' => $caption]);
	//    unlink($argv[1]);
	} catch (\Exception $e) {
	    echo "\n[!] An error has occured when attempting to upload the video file: \n".$e->getMessage()."\n\n";
	    exit(0);
	}
}


if (strpos( $picture, ".jpg") == true) {
    echo "\n[?] Attempting to upload the image file. (.jpg)";
	// Login to the specified account.
	try {
	    $ig->login($username, $password);
	} catch (\Exception $e) {
	    echo "\n[!] An error has occured when attempting to login: ".$e->getMessage()."\n";
	    exit(0);
	}
	try {
	    $photo = new \InstagramAPI\Media\Photo\InstagramPhoto($picture);
	    $ig->timeline->uploadPhoto($photo->getFile(), ['caption' => $caption]);
	//    unlink($argv[1]);
	} catch (\Exception $e) {
	    echo "\n[!] An error has occured when attempting to upload the picture: \n".$e->getMessage()."\n\n";
	    exit(0);
	}
}

if (strpos( $picture, ".png") == true) {
    echo "\n[?] Attempting to upload the image file. (.png)";
	// Login to the specified account.
	try {
	    $ig->login($username, $password);
	} catch (\Exception $e) {
	    echo "\n[!] An error has occured when attempting to login: ".$e->getMessage()."\n";
	    exit(0);
	}
	try {
	    $photo = new \InstagramAPI\Media\Photo\InstagramPhoto($picture);
	    $ig->timeline->uploadPhoto($photo->getFile(), ['caption' => $caption]);
	//    unlink($argv[1]);
	} catch (\Exception $e) {
	    echo "\n[!] An error has occured when attempting to upload the picture: \n".$e->getMessage()."\n\n";
	    exit(0);
	}
}


?>
```

Error Log/var_dump:

"Upload of "image.jpg" failed: All retries have failed."

---

### Describe your issue

So basically uploading anything does not work anymore and when I do it takes a while and then it proceeds to give me the error: "Upload of "image.jpg" failed: All retries have failed."

**Comment by Write** [comment]
Same error here

**Comment by JRoy** [comment]
Instagram is currently offline, not an api issue.

**Comment by Feelingly** [comment]
Update now it's: "InstagramAPI\Response\ActivityNewsResponse: Request failed." 
So this means it's completely down now?
